markets/perps-market/contracts/modules/LiquidationModule.sol

```diff
@@ -190,31 +210,16 @@ contract LiquidationModule is ILiquidationModule, IMarketEvents {
 
             emit PositionLiquidated(
                 runtime.accountId,
-                positionMarketId,
+                runtime.positionMarketId,
                 amountLiquidated,
                 newPositionSize
             );
-
-            // using amountToLiquidate to calculate liquidation reward
-            uint256 liquidationReward = includeFlaggingReward
-                ? PerpsMarketConfiguration.load(positionMarketId).calculateLiquidationReward(
-                    amountLiquidated.mulDecimal(price)
-                )
-                : 0;
-
-            // endorsed liquidators do not get liquidation rewards
-            if (
-                ERC2771Context._msgSender() !=
-                PerpsMarketConfiguration.load(positionMarketId).endorsedLiquidator
-            ) {
-                runtime.totalLiquidationRewards += liquidationReward;
-            }
         }
 
         runtime.totalLiquidationCost =
             KeeperCosts.load().getLiquidateKeeperCosts() +
             costOfFlagExecution;
-        if (runtime.totalLiquidated > 0) {
+        if (includeFlaggingReward || runtime.totalLiquidated > 0) {
```

Comment:
could we rename `includeFlaggingReward` -> `positionFlagged`.  a bit easier to understand imo